proto/pdpb.proto

```diff
@@ -240,6 +240,8 @@ message RegionHeartbeatRequest {
     uint64 keys_read = 9;
     // Approximate region size.
     uint64 approximate_size = 10;
+    // Unix timestamp when send the heartbeat.
+    uint64 timestamp = 11;
```

Comment:
how to handle the time drift between PD and TiKV?

Reply:
E.g, TiKV falls behind PD too much, how do we do.

Reply:
pd just needs to care about the tendency of (time.Now() - timestamp). In the case tikv falls behind pd, we will get a negative value, but if the heartbeat is stall we still can learn it cause the diff value is increasing.